osu.Game.Tests/Visual/Multiplayer/TestSceneMultiplayerPlaylist.cs

```diff
@@ -29,10 +28,13 @@ namespace osu.Game.Tests.Visual.Multiplayer
 {
     public partial class TestSceneMultiplayerPlaylist : MultiplayerTestScene
     {
-        private MultiplayerPlaylist list;
-        private BeatmapManager beatmaps;
-        private BeatmapSetInfo importedSet;
-        private BeatmapInfo importedBeatmap;
+        [Cached(typeof(IBindable<PlaylistItem>))]
+        private readonly Bindable<PlaylistItem> currentItem = new Bindable<PlaylistItem>();
```

Comment:
As mentioned IRL, I'm kinda against having raw `IBindable<>` things being cached like this. I know this is an existing usage, but hoping we can refactor as a future pass. For instance, putting the bindables inside a `MultiplayerState` would resolve the issue for me.

I just really dislike caching of bindables like this (and tracking down where the bindables are cached). It took me quite some minutes to figure out that you weren't adding a new cached bindable here and that it was already being provided in `RoomSubScreen`.

Reply:
Completely agree. The path I'm taking right now is to completely remove bindable caching either directly or ferried via another object, and seeing if a case comes up where it breaks down and the current structure makes more sense (hopefully not).

Basically, I don't know how things will end up yet because I haven't gone the full way 😛 It's entirely possible this PR may be reverted at some point if things don't pan out well.